microovn/ovn/bootstrap.go

```diff
@@ -59,15 +59,15 @@ func Bootstrap(ctx context.Context, s state.State, initConfig map[string]string)
 		return err
 	}
 
-	err = node.EnableService(ctx, s, types.SrvCentral)
+	err = generateEnvironment(ctx, s)
```

Comment:
I don't think I quite understand. Why does this call to `generateEnvironment` fixes this issue, when there's call to this same function 20 lines above? How does enabling `switch` makes a difference?

Reply:
im going back to try figure out the actual reason some processes find the env file empty, unsure as to why